src/IndexRegistry.sol

```diff
@@ -244,65 +302,48 @@ contract IndexRegistry is IndexRegistryStorage {
                                  VIEW FUNCTIONS
     *******************************************************************************/
 
-    /// @notice Returns the _indexToOperatorIdHistory entry for the specified `operatorIndex` and `quorumNumber` at the specified `index`
-    function getOperatorIndexUpdateOfIndexForQuorumAtIndex(uint32 operatorIndex, uint8 quorumNumber, uint32 index) external view returns (OperatorUpdate memory) {
-        return _indexToOperatorIdHistory[quorumNumber][operatorIndex][index];
+    /// @notice Returns the _indexHistory entry for the specified `operatorIndex` and `quorumNumber` at the specified `index`
+    function getOperatorUpdateAtIndex(uint32 operatorIndex, uint8 quorumNumber, uint32 index) external view returns (OperatorUpdate memory) {
+        return _indexHistory[quorumNumber][operatorIndex][index];
```

Comment:
feels weird that these orderings are different (function inputs vs. mapping structure)

Reply:
Agreed. I'm not a fan of `operatorIndex` naming either. IMO the worst part of the `IndexRegistry` is trying to make the implicit structure of the "operator list at block number X" pop out from the code.

Part of the confusion is calling things an "index" sometimes, vs the "operator count" at others. I'm not sure how to marry these two concepts. If you have any ideas I'd love to hear them.

I'll change the param order :)

Reply:
see [231bbe3](https://github.com/Layr-Labs/eigenlayer-middleware/pull/43/commits/231bbe33b677545dbb092033f0be21fdfc83d501)